css/viewer.scss

```diff
@@ -21,6 +21,14 @@
 	background-image: url('../img/x-office-presentation.svg');
 }
 
+.icon-collabora {
+  cursor: pointer;
```

Comment:
This should already be done with https://github.com/nextcloud/server/blob/ee987d74303cb38b864f96660cd2ee6d6552ebfd/core/css/styles.scss#L62
@jancborchardt Right?
@luka-nextcloud Is it not?